src/reference/guide/08-Library-Dependencies.md

```diff
@@ -153,32 +153,26 @@ to ensure binary compatibility.
 
 See [Cross Building][Cross-Build] for some more detail on this.
 
-#### Ivy revisions
+#### Dynamic versions (Discouraged)
 
 The `version` in `organization % moduleName % version` does not have to be a
-single fixed version. Ivy can select the latest revision of a module
-according to constraints you specify. Instead of a fixed revision like
-`"1.6.1"`, you specify `"latest.integration"`, `"2.9.+"`, or `"[1.0,)"`. See the
-[Ivy
-revisions](https://ant.apache.org/ivy/history/2.3.0/ivyfile/dependency.html#revision)
+single fixed version.
+Cousier and Ivy can select the latest revision of a module according to constraints you specify. 
+Instead of a fixed revision like
+`"1.6.1"`, you specify `"latest.integration"`, `"2.9.+"`, or `"[1.0,)"`. 
+These dynamic versions might work one day, but break the next, and make the build non-repeatable.
+
+A Maven version range such as `[1.3.0,)` can be used to specify a dependency,
+and it can appear transitively in existing POM files. When a dependency manager
+like Coursier finds a version range, it will go out to the Internet to find the latest version.
+This could lead to surprising behavior where the effective version changes over time,
+even when there is a specified version of the library within the version range.
```

Comment:
im still confused about the referenced `specified version`, if a range was provided as `version`, where did the `specified version` come from?

Reply:
I guess transitively from other subproject or libraries.

Reply:
Addressed in 55ef0a5.